kqueen/auth/ldap.py

```diff
@@ -15,38 +16,50 @@ def __init__(self, *args, **kwargs):
         """
 
         super(LDAPAuth, self).__init__(*args, **kwargs)
+        if not all(hasattr(self, attr) for attr in ['uri', 'admin_dn', 'password']):
+            logger.error('Failed to configure LDAP synchronization error.')
+            raise ImproperlyConfigured
+
+        # Define Kqueen rdn for all dc's
+        dc_list = []
```

Comment:
Maybe replace this with something like:
```
d_names = ldap.dn.explode_dn(self.admin_dn)
dc_list = [dc for dc in d_names if dc.startswith('dc=')
```
Feels a bit better and I don't really like using single letter variable names outside of generator like this